config/core/resources/cloudauditlogssource.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Synced with @Harwayne offline, we should eventually update the CRDs of all resources to use `apiextensions.k8s.io/v1` as apiVersion, they will be done in a separate PR.